src/main/java/bit/javaoop/FixedBudgetManager.java

```diff
@@ -0,0 +1,23 @@
+package bit.javaoop;
+
+public class FixedBudgetManager extends AbstractManager {
+    public FixedBudgetManager(String name, double salary, double budget) {
+        super(name, salary);
+        this.maxbudget=budget;
+    }
+    private double maxbudget;
+
+    @Override
+    public boolean canhire(AbstractEmployee employee) {
+        return (maxbudget >= getbudget() + employee.getSalary());
+    }
+
+    @Override
+    public void hire(AbstractEmployee employee) {
```

Comment:
Działanie metody sugeruje, że powinna się nazywać `hireIfPossible`, ponieważ jeśli pracownik nie spełni warunku zatrudnienia to metoda nie zrobi nic.